sql/src/test/java/org/apache/druid/sql/calcite/QueryTestRunner.java

```diff
@@ -667,7 +674,9 @@ public QueryTestRunner(QueryTestBuilder builder)
         verifySteps.add(new VerifyNativeQueries(finalExecStep));
       }
       if (builder.expectedResultsVerifier != null) {
-        verifySteps.add(new VerifyResults(finalExecStep));
+        // Don't verify the row signature when MSQ is running, since the broker receives the task id, and the signature
+        // would be {TASK:STRING} instead of the expected results signature
+        verifySteps.add(new VerifyResults(finalExecStep, !config.isRunningMSQ()));
```

Comment:
it might be nice to have something that functions equivalently for msq based tests (if there isn't already) but this doesn't seem like a blocker.

Reply:
Yep, I wanted to separate out the row signature check and the results check, though chose not to keep it in this PR.